core/dbt/config/renderer.py

```diff
@@ -107,7 +107,7 @@ def __init__(
         if cli_vars is None:
             cli_vars = {}
         if profile:
```

Comment:
Still need to pass in profile here since this renderer is being used in integration tests all around the place(dbt-databricks even), hard to do the update.